src/standalone/Snow/Snow.jl

```diff
@@ -63,7 +63,7 @@ function SnowParameters{FT}(
     fS_c = FT(0.2),
     κ_ice = FT(2.21),
     earth_param_set::PSE,
-) where {FT, PSE}
+) where {FT <: AbstractFloat, PSE}
```

Comment:
not sure we need the FT <: AbstractFloat here? https://github.com/CliMA/ClimaLSM.jl/blob/ad31773009c9309d5e45b083603873a8914be286/src/standalone/Snow/Snow.jl#L13